integration/integration_test_utils.py

```diff
@@ -155,12 +178,19 @@ def driver(self, IsAggregationlessDriver=False, *args, **kwargs):
                 key_prefix="storey-test:",
                 **kwargs,
             )
+        elif self.driver_name == "SQLDriver" and IsAggregationlessDriver:
+            sql_driver_class = storey.sql_driver.SQLDriver
+            return sql_driver_class(db_path=SQL_DB, primary_key=primary_key)
+        elif self.driver_name == "SQLDriver" and not IsAggregationlessDriver:
+            pytest.skip("SQLDriver doesnt support aggregation")
         else:
             driver_name = self.driver_name
-            raise ValueError(f'Unsupported driver name "{driver_name}"')
+            raise ValueError(
+                f'Unsupported driver name "{driver_name}" with IsAggregationlessDriver = {IsAggregationlessDriver}'
```

Comment:
Why add this to the error message? Why does it matter?

Reply:
@gtopper because the SQLDdriver  Is aggregation less.